README.md

```diff
@@ -9,9 +9,9 @@ Web client for [`rezervo`](https://github.com/mathiazom/rezervo), including book
 
 #### 🧑‍🔧 Setup
 
-1. Install dependencies using Yarn
+1. Install dependencies using PnPm
```

Comment:
Isn't it just `pnpm`? I see that the website uses random capitalization 🙃, but maybe just keep it simple?

Reply:
Sure, fixed it now. Not sure why I have that capitalization wired into my brain 🤷🏻